app/src/lib/rules/part_of_speech_rules.js

```diff
@@ -275,6 +275,36 @@ const part_of_speech_rules_json = [
 		'remove': 'Adjective',
 		'comment': 'The people were amazed(V/Adj) by Jesus\' words.',
 	},
+	{
+		'name': 'If Adjective-Adposition followed by a determiner, delete the Adjective',
+		'category': 'Adjective|Adposition',
+		'context': {
+			'followedby': { 'tag': 'determiner' },
+		},
+		'remove': 'Adjective',
+		'comment': 'Daniel 1:4 Asphenaz taught them about(Adj/Adp) the ...',
+	},
+	{
+		'name': 'If Adjective-Adposition "about" followed by an Adjective, delete the Adposition',
+		'category': 'Adjective|Adposition',
+		'trigger': { 'stem': 'about' },
+		'context': {
+			'followedby': { 'category': 'Adjective' },
+			'notfollowedby': { 'stem': 'all|many|each' },
+		},
+		'remove': 'Adposition',
+		'comment': "Daniel 2:1  \"for about(Adj/Adp) two years\"  This rule applies specifically to the word \"about\" because in general, both Adjectives and Adpositions can precede Adjectives.  Exclude the word 'all' as in Ruth 3:16 \"Ruth told Naomi about(Adj/Adp) all(Adj) the things that Boaz did.\" Exclude 'many' as in Kande's Story 5:9 \"They talked about many things.\"  This rule really should specify that when 'about' is followed by a numeric Adjective, delete the Adposition.",
```

Comment:
swap those quotes to get rid of the backslashes

Reply:
I copied that comment from TBTA, and it contains both single and double quotes. So either I backslash the double quotes, or I backslash the single quotes. Although I could remove some of them altogether

Reply:
I changed all the string to only contain single quotes, and left the outer string quotes as double.